src/modules/dashboard/components/ColonyHome/ColonyHome.tsx

```diff
@@ -156,17 +111,38 @@ const ColonyHome = ({ match, location }: Props) => {
         <div className={styles.mainContent}>
           <ColonyTotalFunds colony={colony} />
           <Switch>
-            <Route path={COLONY_EVENTS_ROUTE} component={() => <>Events</>} />
             <Route
+              exact
+              path={COLONY_HOME_ROUTE}
+              component={() => <>Actions</>}
+            />
+            <Route
+              exact
+              path={COLONY_EVENTS_ROUTE}
+              component={() => <>Events</>}
+            />
+            <Route
+              exact
               path={COLONY_EXTENSIONS_ROUTE}
-              component={() => <>Extensions</>}
+              render={(props) => (
+                <Extensions {...props} colonyAddress={data.colonyAddress} />
+              )}
+            />
+            <Route
+              exact
+              path={[
+                COLONY_EXTENSION_DETAILS_ROUTE,
+                COLONY_EXTENSION_SETUP_ROUTE,
+              ]}
+              render={(props) => (
+                <ExtensionDetails
+                  {...props}
+                  colonyAddress={data.colonyAddress}
+                />
+              )}
             />
-            <Route path={COLONY_HOME_ROUTE} component={() => <>Actions</>} />
           </Switch>
         </div>
-        <aside className={styles.rightAside}>
```

Comment:
Was the spec that required to display enabled extensions in the sidebar removed?

![Screenshot from 2020-11-17 14-09-03](https://user-images.githubusercontent.com/1193222/99388610-84c0a300-28de-11eb-86ff-a5976142d6cc.png)